Postgresql: Database schema not up to date after installation

waader opened the pull request:
As described in #4922 and #5177 the posgresql database schema is not up to date after installation. Apply this patch before installing. Then install on a postgresql database with "Install Sample Data" set to None. Login in the backend and go to the "Extension Manager" -> "Database" and verify that the database schema is uptodate.

Commit:
'postgres-schema-update'

roland-d commented:
@waader Have you already created the additional patch? Just want to be sure all users can update.

wilsonge commented:
@waader can we have that other PR please. I want all this ready to ship in beta 2 (one week today)

roland-d commented:
@waader @wilsonge Just want to poke you guys so this isn't forgotten.

waader commented:
@roland-d 

I made an update file with this two line:
CREATE INDEX "#__contentitem_tag_map_idx_tag_name" ON "#__contentitem_tag_map" ("tag_id", "type_alias");
ALTER TABLE "#__associations" ALTER COLUMN "id" TYPE int;

But before I applied that file I took an old joomla installation 3.1.1. and updated it to 3.2.7. From there I updated to 3.4.0-beta3 and looked up the database schema. Too my surprise the index was present and the column was of type integer. All without applying my update file. When I go to the extension manager -> database I get the message "Database table structure is up to date".

Maybe there where changes in later version but then my update file could cause problems with versions prior to that changes.

So I don´t know what there should be done.

roland-d commented:
@waader Apologies for my late reply but if all is working as should I think there is nothing else to be done.Thanks.
